poshc2/client/command_handlers/ImplantHandler.py

```diff
@@ -477,6 +507,56 @@ def do_show_urls(user, command):
     clear()
 
 
+def do_show_hosted_files(user, command):
+    files = update_cache_urls()
+    filesformatted = "ID  URI  FilePath  ContentType  Base64  Active\n"
+    for i in files:
+        filesformatted += "%s  %s  %s  %s  %s %s \n" % (i[0], i[1], i[2], i[3], i[4], i[5])
+    print_good(filesformatted)
+    input("Press Enter to continue...")
+    clear()
+
+
+def do_add_hosted_file(user, command):
+    URI = input("URI Path: .e.g. downloads/2020/application.docx: ")
+    FilePath = input("File Path: .e.g. /tmp/application.docx: ")
+    ContentType = input("Content Type: .e.g. (text/html): ")
+    if ContentType == "":
+        ContentType = "text/html"
+    Base64 = no_yes_prompt("Base64 Encode File")
+    if Base64 == False:
```

Comment:
With booleans you don't need to compare to True/False, can just do `if not Base64` or `if Base64`.
I'd still pass around True/False not "Yes"/"No" then in the database store it as a boolean column (or 1/0 in sqlite)